tests/integration/kubernetes/runtimeclass_workloads/pod-trusted-ephemeral-data-storage.yaml

```diff
@@ -0,0 +1,23 @@
+---
+kind: Pod
+apiVersion: v1
+metadata:
+  name: trusted-ephemeral-data-storage
+  annotations:
+    io.katacontainers.config.hypervisor.block_device_driver: virtio-blk
```

Comment:
Do you get a pass on TDX? I reported some problems in #11238 but haven't re-tested it. JFYI

Reply:
Seems TDX failed but it's not a required test: https://github.com/kata-containers/kata-containers/actions/runs/20235432146/job/58093164997?pr=10559 LMK if you'd like me to skip it regardless.